validphys2/src/validphys/checks.py

```diff
@@ -44,8 +44,7 @@ def check_pdf_is_montecarlo(ns, **kwargs):
 
 
 @make_argcheck
-def check_pdf_is_montecarlo_or_symmhessian(ns, **kwargs):
-    pdf = ns['pdf']
+def check_pdf_is_montecarlo_or_symmhessian(pdf, **kwargs):
     etype = pdf.error_type
```

Comment:
These lines might _need_ the fix as well actually. I only looked at the one that was failing.

Reply:
I think they don't because of the use of `make_check` instead of `make_argcheck`. The other check, which is newer, uses `make_argcheck` as recommended by reportengine.

So I'll leave them be (with `ns` and `make_check` for the time being...)

Reply:
That is very likely the case indeed! I am happy with that.